app/src/sharedTest/java/org/oppia/android/app/completedstorylist/CompletedStoryListActivityTest.kt

```diff
@@ -130,6 +138,23 @@ class CompletedStoryListActivityTest {
     ApplicationProvider.getApplicationContext<TestApplication>().inject(this)
   }
 
+  private fun createCompletedStoryListActivityIntent(): Intent {
```

Comment:
There is already one such function available. `createCompletedStoryListActivityIntent` use that directly.

Reply:
Done